admin/admin.go

```diff
@@ -13,45 +15,53 @@ import (
 
 // Server manages InfluxDB's admin web server.
 type Server struct {
-	port     string
+	sync.Mutex
```

Comment:
We shouldn't do it this way, as it promotes it to the public level that end users could now lock and unlock (crazy as it sounds).